internal/web3ext/web3ext.go

```diff
@@ -822,6 +838,22 @@ web3._extend({
 			name: 'serverInfo',
 			getter: 'les_serverInfo'
 		}),
+		new web3._extend.Property({
+			name: 'gatewayFee',
+			getter: 'les_gatewayFee'
+		}),
+		new web3._extend.Property({
+			name: 'suggestGatewayFee',
+			getter: 'les_suggestGatewayFee'
+		}),
```

Comment:
I would recommend moving this from properties to methods. Because the suggestion could conceivable be computationally non-trivial or return a non-deterministic result, it would be better to use the semantics of calling a method rather than treating it as a property access.